packages/frontend/app/components/user-menu.js

```diff
@@ -49,30 +53,29 @@ export default class UserMenuComponent extends Component {
     return true;
   }
 
-  handleArrowDown(evt, item) {
-    if (evt.target.tagName.toLowerCase() === 'button') {
+  handleArrowDown(event, item) {
+    if (event.target.tagName.toLowerCase() === 'button') {
       this.isOpen = true;
+      setTimeout(
+        () => event.target.parentElement.querySelector('.menu a:first-of-type').focus(),
+        1,
+      );
     } else {
       if (item.nextElementSibling) {
         item.nextElementSibling.querySelector('a').focus();
       } else {
-        // eslint-disable-next-line ember/no-runloop
-        schedule('afterRender', () => {
-          this.element.querySelector('.menu li:nth-of-type(1) a').focus();
-        });
+        item.parentElement.firstElementChild.querySelector('a').focus();
       }
     }
   }
 
   handleArrowUp(item) {
-    if (item.previousElementSibling) {
-      item.previousElementSibling.querySelector('a').focus();
-    } else {
-      this.element.querySelector('.menu li:last-of-type a').focus();
+    if (item) {
```

Comment:
I'm ok with this. Could also use the `item?.previousIndex...` optional chaining syntax which would do the same thing with slightly less indentation.

Reply:
I didn't like it when I wrote it, so thank you for reminding me of the `?` optional thing!